manifests/webhook/config.pp

```diff
@@ -44,6 +44,7 @@
   $configfile_symlink         = '/etc/webhook.yaml',
   $enable_mutex_lock          = $r10k::params::webhook_enable_mutex_lock,
   Array $ignore_environments  = $r10k::params::webhook_ignore_environments,
+  $mco_arguments              = $r10k::params::webhook_mco_arguments,
```

Comment:
Well, if user didn't specified this parameter then default `undef` will be used so I guess it should be `Variant[String, Undef]` ?